pkg/pipeline/extract/conntrack/conn.go

```diff
@@ -100,7 +101,9 @@ func (c *connType) getNextHeartbeatTime() time.Time {
 func (c *connType) toGenericMap() config.GenericMap {
 	gm := config.GenericMap{}
 	for k, v := range c.aggFields {
-		gm[k] = v
+		if v != nil && (reflect.TypeOf(v).Kind() != reflect.Float64 || v.(float64) != 0) {
```

Comment:
should this be && i.e
```golang
if v != nil && reflect.TypeOf(v).Kind() == reflect.Float64 && v.(float64) != 0 {
```

Reply:
`aggFields` can be other types than `float64` here
Example: `string` for `PktDropLatestState` or `PktDropLatestDropCause` on `last` aggregate

Reply:
/lgtm